src/zm_monitor.cpp

```diff
@@ -138,7 +138,10 @@ bool Monitor::MonitorLink::connect() {
 
     Debug(1, "link.mem.size=%d", mem_size);
 #if ZM_MEM_MAPPED
-    map_fd = open( mem_file, O_RDWR, (mode_t)0600 );
+    /* mode 660 to allow group www-data */
+    mode_t oldmask = umask(0002);
```

Comment:
Here we are just opening the file, not creating it, so do we need to change the umask here?

Reply:
I think the mode being passed in open is ignored as well

Reply:
Agreed. I can remember i just focused on 600 and a "shouldn't harm". Regarding `man 2 open` it gets ignored.